Makefile.js

```diff
@@ -159,7 +159,8 @@ function execSilent(cmd) {
  * @private
  */
 function generateBlogPost(releaseInfo) {
-    const output = ejs.render(cat("./templates/blogpost.md.ejs"), releaseInfo),
+    const ruleList = ls("lib/rules").map(ruleFileName => ruleFileName.replace(/\.js$/, "")).sort((ruleA, ruleB) => ruleB.length - ruleA.length);
```

Comment:
Sure, I'll put it on multiple lines.

You're correct about the reason for sorting. For example, the string `no-undefined` should get matched to the rule `no-undefined`, rather than the rule `no-undef` followed by the string `ined`.